pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -665,36 +665,6 @@ cdef class _Timedelta(timedelta):
         else:
             return "D"
 
-    @property
-    def days(self):
```

Comment:
technically then won't ``self._sign`` be wrong (not that we are actually using it here)? I guess this is ok

Reply:
The only place where self._sign gets used is in _repr_base, which calls _ensure_components at the start of the method to set self._sign. That happens irrespective of the property definitions. 

If _repr_base were ever re-factored in the future it could definitely be missed. OK to open a separate issue to see if we can refactor the _repr_base call and make it more "future-proof" by removing the need to self._sign altogether?

FWIW I checked the test cases and didn't see one that explicitly handles the repr of negative Timedeltas, so I'll add one and re-push

Reply:
Scratch my comment on not having test cases - you'll see some in test_arithmetic that ensure we have the appropriate Timedelta repr, so I'm not planning to re-push unless you have additional changes you'd like.

https://github.com/pandas-dev/pandas/blob/a172ff9dd32d7f6ba077c82b6cf6a31dcc7f2b45/pandas/tests/indexes/timedeltas/test_arithmetic.py#L235